app/src/main/java/org/gdg/frisbee/android/api/PlusImageResizer.java

```diff
@@ -0,0 +1,41 @@
+package org.gdg.frisbee.android.api;
+
+import org.gdg.frisbee.android.api.model.plus.ImageInfo;
+import org.gdg.frisbee.android.app.App;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class PlusImageResizer implements Interceptor {
```

Comment:
This gives size to the api but the intention of this is not to resize the image. It gets the image from Google+ URL.